pandas-stubs/core/dtypes/generic.pyi

```diff
@@ -8,24 +8,13 @@ class ABCFloat64Index: ...
 class ABCMultiIndex: ...
 class ABCDatetimeIndex: ...
 class ABCTimedeltaIndex: ...
-class ABCPeriodIndex: ...
```

Comment:
I"m guessing all the `ABC` classes here could be deleted, and if we later import them, we could get rid of the imports, since the classes are empty